src/components/Discussion/Internal/Comment/Actions.js

```diff
@@ -71,18 +64,6 @@ const styles = {
       flexShrink: 0
     }
   }),
-  selectedVoteButton: css({
-    lineHeight: 1,
-    fontSize: '24px',
-    textAlign: 'center',
-    height: '40px',
-    margin: 0,
-    color: colors.primary,
-    '& > svg': {
-      display: 'block',
-      flexShrink: 0
-    }
-  }),
```

Comment:
This style needs adequate replacement:
<img width="1885" alt="Screenshot 2020-11-03 at 00 36 55" src="https://user-images.githubusercontent.com/410211/97930893-c0029400-1d6c-11eb-9ba3-35c46997895d.png">